fem/bilininteg.cpp

```diff
@@ -2162,36 +2162,29 @@ void VectorDiffusionIntegrator::AssembleElementMatrix(
       }
    }
 
-   elmat = 0.0;
```

Comment:
We should probably still initialize `elmat` to zero so ensure that its off-diagonal bocks are properly initialized.

Reply:
We've switched from an `+=` to an `=` for the last step where `elmat` is assigned: that's why I removed it. I think we should fill all of it -- if I'm not missing something.

Reply:
True but you only fill `sdim` blocks along the diagonal which each have a size of `dof x dof`.  So, in 2D you fill 2 blocks on the diagonal and this leaves 2 off-diagonal blocks of size `dof x dof` untouched, in 3D there are 3 filled blocks and 6 untouched blocks.  To put it another way the nested loop only makes `sdim*dof*dof` assignments but there are `sdim*sdim*dof*dof` entries.

Reply:
Ok, thanks -- I'll put it back.

Reply:
`done`